app/models/user.rb

```diff
@@ -1,16 +1,38 @@
 class User < ActiveRecord::Base
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  require 'rest_client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.